### DISCUSSION TITLE
addChannel 사용 방법을 잘 모르겠습니다.

### DISCUSSION THREAD

**Discussion by h2saj5xz** [storycraft/node-kakao#235] [Q&A] [answered]
안녕하세요.

최근 업데이트로 100개 이상의 채널리스트에서 
챗이 올때 채널 추가가 잘 되고 있습니다. 
감사합니다.

다른글에서 TalkChannelList 객체의 open, normal 속성에서 
addChannel 메서드를 사용해 특정 id 채널을 추가할 수 있다고 하셨는데
사용방법을 잘 모르겠습니다.

추가하려는 채널은 이전에 챗이 왔을때 보존해준 채널ID가 있고
클라이언트 접속시 채널이 100개가 넘어 불러오지 못한 채널입니다.
챗이 오지않아도 보존해둔 채널 ID로 채널을 추가하고 싶습니다.

제가 사용해본 코드와 결과는 다음과 같습니다.
normal, open 모두 같은 결과가 옵니다.

client.channelList.size;
-> 100

client.channelList.normal.addChannel('12345');
-> TypeError: Cannot read property 'toString' of undefined

talkClient.channelList.normal.addChannel({channelId: '12345'});
-> TypeError: Cannot read property 'push' of undefined

**Comment by cjh980402** [comment, marked as answer]
talkClient.channelList.normal.addChannel({channelId: '12345'});
여기서 채널 ID를 문자열 대신 Long객체를 생성하고 넘겨보세여.

**Reply by h2saj5xz** [reply]
제가 TS를 안써봐서 JS를 쓰고있는데, Long타입으로 주고받는 방법이 있나요?
그동안 Long타입이 없으니 문자열로 주고받고 했었거든요.

**Reply by cjh980402** [reply]
노카에서 기본적으로 Long타입을 export해줘서 그걸로 쓰시면 돼요.

**Reply by h2saj5xz** [reply]
오 노드카카오에 Long 객체가 있었네요. 
감사합니다!
